dev-libs/quazip/quazip-0.7.6.recipe

```diff
@@ -70,6 +71,14 @@ INSTALL()
 	mv $prefix/include/* $includeDir
 	rm -rf $prefix/include
 
+	# move the libraries to the correct location
+	if [ -n "$secondaryArchSuffix" ]; then
+		mv $prefix/lib $prefix/lib2
+		mkdir -p $libDir
+		mv $prefix/lib2/* $libDir/
+		rmdir $prefix/lib2
+	fi
```

Comment:
I think instead of adding hacks like this we should patch the `CMakeLists.txt` to use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) then upstream it.

Reply:
If you could point to some examples maybe I could look into it :)

Reply:
Ok, with a small patch the libs are ok, couldn't find a place where to change the destination for the headers yet

Reply:
[Here](https://github.com/stachenov/quazip/blob/d5eaa8be998da2430ffd43e0eefa617cf1550ca3/quazip/CMakeLists.txt#L31) you go :) It should be set to `${CMAKE_INSTALL_INCLUDEDIR}/quazip`.